app/models/bulkrax/csv_collection_entry.rb

```diff
@@ -9,7 +9,7 @@ def factory_class
     # Use identifier set by CsvParser#unique_collection_identifier, which falls back
     # on the Collection's first title if record[source_identifier] is not present
     def add_identifier
-      self.parsed_metadata[work_identifier] = self.identifier
+      self.parsed_metadata[work_identifier] = [self.identifier]
```

Comment:
I haven't run into any issues with this line, which means my `self.identifier` is probably already an array. Can you add `.flatten` to the end of this line just so we don't end up with a nested array please?